azure-pipelines.yml

```diff
@@ -8,16 +8,7 @@ variables:
   GOPATH: '$(Agent.HomeDirectory)/go'
 
 stages:
-  - stage: LintAndBuild
-    jobs:
-      - template: .azure-pipelines/lint_and_build.yml
-
-  - stage: LiveTest
+  - stage: Lint_Build_LiveTest
```

Comment:
Please do not remove the first stage, otherwise there will be nothing run on PR, as you can see in this PR.

Reply:
recoveried